contracts/axelarnet-gateway/tests/execute.rs

```diff
@@ -171,18 +165,18 @@ fn route_to_router_after_contract_call_with_tempered_data_fails() {
     rand::thread_rng().fill_bytes(&mut tx_hash);
     let nonce = rand::random();
 
-    let mut deps = mock_dependencies();
+    let mut deps = mock_axelar_dependencies();
     deps.querier = deps
         .querier
-        .with_custom_handler(reply_with_tx_hash_and_nonce(tx_hash, nonce));
+        .with_custom_handler(axelar_query_handler(tx_hash, nonce, false));
 
     let destination_chain = "destination-chain".parse().unwrap();
     let destination_address = "destination-address".parse().unwrap();
     let payload = vec![1, 2, 3].into();
 
-    utils::instantiate_contract(deps.as_mut()).unwrap();
+    utils::instantiate_contract(emptying_deps_mut(&mut deps.as_mut())).unwrap();
```

Comment:
Assign a variable to the emptying deps instead of having to convert it everywhere

Reply:
I also don't like that, but the function consumes the deps mut